pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -886,7 +886,7 @@ func simplePipelineWithSpecAndParam(pt v1beta1.ParamType) *v1beta1.Pipeline {
 	return p
 }
 
-func simplePipelineWithSpecParamAndKind(pt v1beta1.ParamType, tk v1beta1.TaskKind) *v1beta1.Pipeline {
+func simplePipelineWithSpecParamAndKind(_ v1beta1.ParamType, _ v1beta1.TaskKind) *v1beta1.Pipeline {
```

Comment:
shall we just cleanup the inputs, or more thoroughly the entire helper func which has only been referenced once?

Reply:
OK, I'll handle it.